RUNNING.md

```diff
@@ -91,6 +91,14 @@ Addon-operator expects that "helm" binary is available in $PATH. It detects Helm
 
 **HELM3** — set to "yes" to disable auto-detection and explicitly enable compatibility with helm3.
 
+**HELM_IGNORE_RELEASE** — a name of the release that should not be treated as the module's release. Prevent self-destruction when addon-operator release is stored in the same namespace as releases for modules.
```

Comment:
```suggestion
**HELM3_IGNORE_RELEASE** — a name of the release that should not be treated as the module's release. Prevent self-destruction when addon-operator release is stored in the same namespace as releases for modules.
```
?

Reply:
Yeah, it is implemented only for Helm3 now, but I think it will affect future versions. I wish that we don't need that "ignored release" at all. There should be a mechanism to filter only "our" releases with labels or prefixes.

Helm 2 setup is affected if the cluster-wide tiller's namespace is equal to ADDON_OPERATOR_NAMESPACE. No one stops you from deploying an addon-operator to the same namespace as the tiller. Also, there are situations with "external" tiller, e.g. `werf`.

Reply:
We decided not to rename it.